ISISComputingGroup/ibex_gui
# Ticket 6414: Allow arbitrary position

### Description of work

(Note: contains changes from https://github.com/ISISComputingGroup/IBEX/issues/6409 so merge that first)

See https://github.com/ISISComputingGroup/IBEX/issues/6414

To test:
* With https://github.com/ISISComputingGroup/EPICS-motionSetPoints/pull/21 pulled and made start the samplechanger IOC_test_framework test
* Set an arbitrary position for x/y and confirm the motor move there
* Set a specific named position and confirm the motor moves there

---

#### Code Review


### Final Steps
